Bearded.Utilities/Linq/Extensions.cs

```diff
@@ -234,7 +234,7 @@ public static T RandomElement<T>(this IEnumerable<T> source, Random random)
             {
                 throw new InvalidOperationException("Sequence was empty.");
             }
-            return current;
+            return current!;
```

Comment:
The above check on `count` ensures that this will never be null (unless in fact the sequence contained a null value, in which case that is the intended behaviour).